git/remote.go

```diff
@@ -100,3 +100,14 @@ func AddRemote(name, u string) (*Remote, error) {
 		PushURL:  urlParsed,
 	}, nil
 }
+
+//CheckRemoteExist it's will verify with there is a remote configured at directory
+func CheckRemoteExist() (bool, error) {
+	out, err := exec.Command("git", "remote", "-v").Output()
```

Comment:
In addition to the renaming and function signature I requested above, this function should make use of `GitCommand` and `run.PepareCmd`. For an example you can look at `CurrentBranch`.